### DISCUSSION TITLE
Beta release notes

### DISCUSSION THREAD

**Discussion by nollinvoyd** [aclap-dev/video-downloadhelper#897] [Help and Q&A]
### Your question

Is there somewhere I can go to find this information?  Forgive me if I've mentioned this before. 

I just installed the latest beta.  Unlike most other Firefox addons, there is no Release Notes tab or link to a changelog on it's about:addons page.  

Mainly, any time there is a new release, I want  to learn/educate myself about the changes that will make a more informed, productive user, particularly if there were any settings changes that I may want or need to adjust.  

### Addon & CoApp version and details

Video DownloadHelper 9.0.2.12a1 (beta) 

Video DownloadHelper Companion App 2.0.19

**Comment by nollinvoyd** [comment]
Maybe just a yes or no?

**Comment by jcfrog** [comment]
Hi, there are info here https://addons.mozilla.org/en-US/firefox/addon/video-downloadhelper/versions/

**Reply by nollinvoyd** [reply]
Does that apply that to the beta as well?

**Comment by GrampaWildWilly** [comment]
Documentation in general has not been a priority with this product.  It's so bad, I invented a user manual a few years ago back on our former Google Groups forum (which is back on the air for the time being but there's no telling for how long).  And I'm just another user.  There seems to be no appreciation for the necessity of telling us what's new in the product.  This is something that needs to be provided with EVERY release, including betas.  I don't care if there's only 1 change in the release.  Write a release notes of one line.  Just do it.  Don't whine about not having time.  Do it.  Any feature simply doesn't exist until you document it.  You went to the trouble of putting it in the product.  Tell us about it.

**Comment by paulrouget** [comment]
> Is there somewhere I can go to find this information? 

Nope. Sorry. I'm trying to release notes here sometimes, but 99% of the updates are just about fixing broken websites.

**Reply by GrampaWildWilly** [reply]
Then issue a Release Notes saying that.  List the web sites you fixed.  It's never a long list, is it?  Just spend the 3 minutes it takes & write Release Notes.  Like pretty much every other software product on the planet.

**Reply by nollinvoyd** [reply]
> > Is there somewhere I can go to find this information?
> 
> Nope. Sorry. I'm trying to release notes here sometimes, but 99% of the updates are just about fixing broken websites.

Your time budgeting choices are none of my business.  My initial impression is if you took the opportunity to establish some sort of Release Notes/Changelog here, in the long run it might, in the least, allow some users to figure out some, often simple, questions out by themselves rather than diverting your attention spent responding to their inquiries. 

You say "99% of the updates are just about fixing broken websites".  Then "Fixed broken website" along with whatever your may choose to add as an explanation is all you need.  If I saw that, threads like this wouldn't exist.

On the main discussion page, we have two "Announcement" tabs.  Each has only one thread, and both are a month old.  One actually starts off like a changelog.  You could convert one of them, still call it Announcements/Changelog, whatever you wish, but no discussion, just latest versions news.  If someone wants to discuss things, there's already a place for that.

Or, perhaps a Release Notes link directly above the discussion list.  

Or why not just establish a locked Release Notes sticky thread.  Right there, the first thing at the beginning for every curiosity seeker to study before moving on.

Not criticism.  Hopefully a few helpful suggestions.  I love VDH.

**Comment by GrampaWildWilly** [comment]
Here's my suggestion.  Create a chapter in the Wiki with the title "Release Notes" or "Change Log" or "New Release Announcement" or some other good name you like.  This will be a simple text file.  Every time you change something, add lines to the file according to this pattern:

9.0.2.12a2 2024/7/1 Allowed recognition of non-standard media on missav, javhub
9.0.3.1a1 2024/7/10 Fixed blocking of new download request when Max concurrent downloads = 1
9.0.4.17a4 2024/8/9 Ported old Smartnaming functionality forward
9.1.6.2a7 2024/10/4 Fixed issue of browser unilaterally auto-updating VDH despite a download being in progress
10.0.0.0 2025/5/17 Added support for subtitles

I can imagine cases in which you would do something like this:

9.0.5.0a0 (Firefox only, Windows only) 2024/12/2 Whatever
9.0.5.0a1 (Firefox only, Mac & Linux only) 2024/12/3 Whatever
9.0.5.0a2 (Chrome only) 2024/12/4 Whatever
9.0.5.0a3 (Edge only) 2024/12/6 Whatever

I'm thinking of a case of some problem that you fix in stages, rolling it out on the different platform configurations over a span of days.

This proposed addition to the Wiki can just be a running history of the features & fixes you add to the product.  It's not worth it to retroactively create this for our current 9.0.2.12a1, but surely going forward 1 or 2 lines of this nature each time you post a new release, whether beta or general availability, wouldn't be too painful to do.  Changes to this file would also serve as an announcement of a new release for those of us who understand why permitting automatic updates is a bad idea . . . for the moment.

**Reply by GrampaWildWilly** [reply]
I'm also imagining something like this:

9.4.0.0 2024/12/31 General availability release, same as 9.3.17.42a19 beta
